### ISSUE TITLE
Change address: Create react widgets

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#10853]
## Issue Description

We need to create react widgets that we'll add to this page: https://www.va.gov/change-address/

The user will receive 1 of the 3 states below and then will go to their va.gov profile.

#### copy

**Please sign in to change your address**

Try signing in with your **DS Logon**, **My HealtheVet**, or **ID.me** account. If you don’t have any of these accounts, you can create one now.

[button] **Sign in or create account**

#### screenshot

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/9823d52b-4641-407c-af09-a07a35e29776)

#### copy

**Please verify your identity to change your address**

We need to make sure you’re you—and not someone pretending to be you—before we can give you access to your personal and health-related information. This helps us keep your information safe and prevent fraud and identity theft.

**This one-time process takes about 5 to 10 minutes.**

[button] **Verify your identity**

#### screenshot
(please follow the em dashes with no spaces approach in the copy, not the screenshot)

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/f3f8a03e-cca6-4956-867f-5e531e5a99af)

#### copy

**Go to your VA.gov profile to change your address**

You’ll find your mailing and home addresses in your profile's **Personal and contact information** section.

[button] **Go to your VA.gov profile**

#### screenshot

![image.png](https://images.zenhubusercontent.com/5e8b13016a4378a604f5e799/0039ea21-6f1f-404e-bb67-c2d73919eeef)

---
## Tasks
- [ ] Create react widgets

## Acceptance Criteria
- [ ] React widgets are ready to be added to Drupal page
---
## How to configure this issue
- [x] **Attached to a Milestone** (when will this be completed?)
- [x] **Attached to an Epic** (what body of work is this a part of?)
- [x] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [x] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by DanielleThierryUSDSVA** [comment]
Hey, @kelsonic! This is the issue we talked about for react widgets. Not urgent - shooting for August. So I put it in Sprint 26, but feel free to pick up anytime you have time. And just let me know if you have any questions. Thanks!

FYSA @MickinSahni

**Comment by MickinSahni** [comment]
pulling this out until we can complete another ticket we added to sprint 26 (a form detail copy issue)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @kelsonic @MickinSahni. Realized this got pulled out of sprint 26, but not added back to another sprint. Could you let me know timing ETA for this?

**Comment by Samara-Strauss** [comment]
The last widget ("go to your profile") this needs to be coded in a way that accounts for the fact that we're doing a phased rollout and we'll have two versions of the profile live at the same time until we launch profile 2.0 to 100% of users.

Why this matters — eventually, the "Go to your profile" CTA should link to https://www.va.gov/profile/personal-information. However, while profile 1.0 is still partially live in the phased rollout, the "Go to profile" link can't go to https://www.va.gov/profile/personal-information because that URL does not exist on profile 1.0.

Instead, I would recommend the "Go to your profile" CTA link to https://www.va.gov/profile/ for now until we rollout profile 2.0 to 100% of users. This redirects automatically to https://www.va.gov/profile/personal-information, so it will work for both profile 1.0 and 2.0 while we have both profiles active in the phased rollout. Once we are out to 100% of users, we will change the URL to https://www.va.gov/profile/personal-information as is outlined in #12100.

Let me know if you have any questions about this!

**Comment by DanielleThierryUSDSVA** [comment]
Thanks, @Samara-Strauss! 

@ncksllvn @MickinSahni Per @Samara-Strauss's comment in this ticket (https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10857), the team is targeting end of August for this launch. Could we have react app ready in enough time for us to then update content page (which shouldn't take long at all) for end of August?

**Comment by ncksllvn** [comment]
@MickinSahni I don't know if I'll finish but I'd like to grab this one into this current sprint unless there is something more pressing

**Comment by ncksllvn** [comment]
@DanielleSoCompany Currently the message for verifying your identity is shared across several call-to-action widgets like this one. This means the best and easiest way to update the message is to do it site-widget. Currently, the message reads -

Title: `Please verify your identity to {name of service such as "change your address"}`
Body: `We take your privacy seriously, and we’re committed to protecting your information. You’ll need to verify your identity before we can give you access to your personal health information.`
Button text: `Verify your identity`

**Comment by ncksllvn** [comment]
Also, just noticed that the direct deposit CTA widget has the button text "Go to your profile" rather than "Go to your VA.gov profile". We may want to decide on one to be consistent.

**Comment by ncksllvn** [comment]
Opened this PR, https://github.com/department-of-veterans-affairs/vets-website/pull/13804, which includes the following changes -

1. Implement the Change Address widget, including the routing based on the Flipper toggle
1. Updates the text for the Verify state (site-wide)
1. Updates the text for the Sign-In state (site-wide; just adds the word "now" at the end)
 
LMK if that all sounds good.

**Comment by MickinSahni** [comment]
> @MickinSahni I don't know if I'll finish but I'd like to grab this one into this current sprint unless there is something more pressing

thanks @ncksllvn. keeping it with next sprint given response time/pr review etc.

**Comment by ncksllvn** [comment]
This is merged and will be available in Drupal today as a React widget. It is a React Widget -> CTA (call-to-action) - with app ID `change-address`

**Comment by MickinSahni** [comment]
> This is merged and will be available in Drupal today as a React widget. It is a React Widget -> CTA (call-to-action) - with app ID `change-address`

hey nick, can you confirm this is available? want to see if we can unblock [Change address: Update Drupal page #10857](#10857) for @RLHecht and @DanielleThierryUSDSVA
